staging/src/k8s.io/apiserver/pkg/storage/cacher/testing/cacher_whitebox_test.go

```diff
@@ -0,0 +1,213 @@
+/*
```

Comment:
let's rename this directory from cacher/testing to cacher/tests/

[testing suggests that it's testing utilities, but here we have actual tests.]

Reply:
BTW - I don't like the fact of testing stuff in different directory compare to where the implementation really is..
[I'm assuming that we can't have different package within the same directory, right?]

So we should at least add a comment that it tests what's in cacher/ directory...